core/src/syn/error/render.rs

```diff
@@ -170,6 +170,11 @@ impl fmt::Display for Snippet {
 	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
 		// extra spacing for the line number
 		let spacing = self.location.line.ilog10() as usize + 1;
+		for _ in 0..spacing {
+			f.write_str(" ")?;
+		}
```

Comment:
```suggestion
		f.write_str(&" ".repeat(spacing))?;
```

Reply:
I like just looping over the write_str more as it doesn't allocate a string.